src/main/memorygame/app.cljs

```diff
@@ -36,47 +36,71 @@
              (and (= kort (@tilstand :kort)) (not (= placering (@tilstand :placering))))
              (do
-               (swap! tilstand update-in [(if
-                                            (even? (@tilstand :næstespiller)) :player1 :player2) :points] inc)
+               (println @tilstand)
+               (println "Før letten")
+               (let [nuværendespiller
+                     (@tilstand :nuværendespiller)]
+                    (println nuværendespiller)
+                    (swap! tilstand update-in [nuværendespiller :points] inc)
+                    )
+               (println @tilstand)
                (css/add-class! elm "flip-card-hidden")
-               (css/add-class! (-> js/document (.getElementById (@tilstand :placering))) "flip-card-hidden"))
+               (css/add-class! (-> js/document (.getElementById (@tilstand :placering))) "flip-card-hidden")
+
+               (swap! tilstand assoc-in [:kort] nil)
+               (swap! tilstand assoc-in [:placering] nil)
+               (println @tilstand)
+
+               ; tjek om spillet er slut.
+
+               )
 
              ;  (hvis kortene ikke er ens, vend bagsiden opad paa begge kort, skift spilleren)
              :else
              (do
+               ; Wait five seconds so we can see the picture
+
                (css/remove-class! elm "flip-card-transform")
                (css/remove-class! (-> js/document (.getElementById (@tilstand :placering))) "flip-card-transform")
-               (swap! tilstand update-in [:næstespiller] inc)
+               (let [
+                     nuværendespiller (@tilstand :nuværendespiller)
+                     swappedspiller (#(if (= % :player1) :player2 :player1) nuværendespiller)
+                     ]
+
+                    (swap! tilstand assoc-in [:nuværendespiller] swappedspiller))
+
                (swap! tilstand assoc-in [:placering] nil)
-               (swap! tilstand assoc-in [:kort] nil)))
-           ; tjek om spillet er slut.
+               (swap! tilstand assoc-in [:kort] nil)))))
+
 
 
-           ;(cond
-           ;  (first click, kort gemt i tilstand og kortet forsiden op)
-           ;
-           ;  (hvis kortene er ens, slet kortene og giv point til spileren, tjek om spillet er slut (giv match point), spilleren fortsaetter)
-           ;
-           ;  (hvis kortene ikke er ens, vend bagsiden opad paa begge kort, skift spilleren))
+;(cond
+;  (first click, kort gemt i tilstand og kortet forsiden op)
+;
+;  (hvis kortene er ens, slet kortene og giv point til spileren, tjek om spillet er slut (giv match point), spilleren fortsaetter)
+;
+;  (hvis kortene ikke er ens, vend bagsiden opad paa begge kort, skift spilleren))
 
 
 (defn kort [xs]
       (let [element (str "felt-" (first xs))]
 
-           [:td {:on-click #(spil xs element)}
+           [:td {:on-click #(spil xs element) :id (str "td-" (first xs))}
```

Comment:
React brokker sig, når der er lister uden unikke id'er

Reply:
good to know